applications/vanilla/controllers/class.categoriescontroller.php

```diff
@@ -455,6 +456,7 @@ public function all($Category = '', $displayAs = '') {
         $this->addModule('DiscussionFilterModule');
         $this->addModule('BookmarkedModule');
         $this->addModule($CategoryFollowToggleModule);
+        $this->addModule('TagModule');
```

Comment:
The module does nothing is the plugin is not enabled. See https://github.com/vanilla/vanilla/pull/5049#issuecomment-275528636